vm.go

```diff
@@ -542,6 +548,7 @@ func (s *stash) deleteBinding(name unistring.String) {
 func (vm *vm) newStash() {
 	vm.stash = &stash{
 		outer: vm.stash,
+		vm:    vm, // TODO fix
```

Comment:
Is it expected?

Reply:
I've made a better note for future refactoring